# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1514888076 21600
#      Tue Jan 02 04:14:36 2018 -0600
# Node ID dc25a0dc780045a754266c54727e21b64894a690
# Parent  9d0522881498929857d5abdfb29b534246be1c7a
Correct a host of small formatting issues with Texinfo manual (bug #52774)

basics.txi: Move all of the Octave command line option references that begin
  with a hyphen to underneath the concept-index entry "options, Octave command"
  using the @sortas command, leaving some indentation space with @ @ .  Add
  a missing @cindex reference for the --eval option and the -W option.  Remove
  the deprecated -? help option.  Move the ~/.octaverc and .octaverc index
  entries to "octaverc" via @sortas.  Add an index entry for startup.m.  Move
  the ~/.inputrc index entry to "inputrc" via @sortas.  Move the #! index
  entry from concept index to operator index, qualifying with "self-contained
  script".  Move the #, %, #{ and %{ index entries to the operator index
  qualifying with "comment marker" or "block comment marker" accordingly.

expr.tex: Move the ":end" index entry to "end" via the @sortas command and
  include "end:" as well.

stmt.txi: Change ... and \ index references from @cindex to @opindex and
  position near paragraph reference.

xticks.m: Fix a set of three bogus index entries by supplying a missing field
  with {} for examples in which the function does not return output.
xticklabels.m: Ditto.
yticks.m: Ditto.
yticklabels.m: Ditto.
zticks.m: Ditto.
zticklabels.m: Ditto.
rticks.m: Ditto.
thetaticks.m: Ditto.

diff --git a/doc/interpreter/basics.txi b/doc/interpreter/basics.txi
--- a/doc/interpreter/basics.txi
+++ b/doc/interpreter/basics.txi
@@ -64,183 +64,186 @@ shorter equivalent).
 
 Here is a complete list of the command line options that Octave
 accepts.
 
 
 @table @code
 
 @item --built-in-docstrings-file @var{filename}
-@cindex @code{--built-in-docstrings-file @var{filename}}
+@cindex @sortas{options, Octave command --built-in-docstrings-file} @ @ @code{--built-in-docstrings-file @var{filename}}
 Specify the name of the file containing documentation strings for the
 built-in functions of Octave.  This value is normally correct and should
 only need to specified in extraordinary situations.
 
 @item  --debug
 @itemx -d
-@cindex @code{--debug}
-@cindex @code{-d}
+@cindex @sortas{options, Octave command --debug} @ @ @code{--debug}
+@cindex @sortas{options, Octave command -d} @ @ @code{-d}
 Enter parser debugging mode.  Using this option will cause Octave's
 parser to print a lot of information about the commands it reads, and is
 probably only useful if you are actually trying to debug the parser.
 
 @item --debug-jit
-@cindex @code{--debug-jit}
+@cindex @sortas{options, Octave command --debug-jit} @ @ @code{--debug-jit}
 Enable JIT compiler debugging and tracing.
 
 @item --doc-cache-file @var{filename}
-@cindex @code{--doc-cache-file @var{filename}}
+@cindex @sortas{options, Octave command --doc-cache-file} @ @ @code{--doc-cache-file @var{filename}}
 Specify the name of the doc cache file to use.  The value of @var{filename}
 specified on the command line will override any value of
 @w{@env{OCTAVE_DOC_CACHE_FILE}} found in the environment, but not any commands
 in the system or user startup files that use the @code{doc_cache_file}
 function.
 
 @item  --echo-commands
 @itemx -x
-@cindex @code{--echo-commands}
-@cindex @code{-x}
+@cindex @sortas{options, Octave command --echo-commands} @ @ @code{--echo-commands}
+@cindex @sortas{options, Octave command -x} @ @ @code{-x}
 Echo commands as they are executed.
 
 @item --eval @var{code}
+@cindex @sortas{options, Octave command --eval} @ @ @code{--eval @var{code}}
 Evaluate @var{code} and exit when finished unless @option{--persist} is also
 specified.
 
 @item --exec-path @var{path}
-@cindex @code{--exec-path @var{path}}
+@cindex @sortas{options, Octave command --exec-path} @ @ @code{--exec-path @var{path}}
 Specify the path to search for programs to run.  The value of @var{path}
 specified on the command line will override any value of
 @w{@env{OCTAVE_EXEC_PATH}} found in the environment, but not any commands
 in the system or user startup files that set the built-in variable
 @w{@env{EXEC_PATH}}.
 
 @item --force-gui
-@cindex @code{--force-gui}
+@cindex @sortas{options, Octave command --force-gui} @ @ @code{--force-gui}
 Force the graphical user interface (GUI) to start.
 
 @item  --help
 @itemx -h
-@itemx -?
-@cindex @code{--help}
-@cindex @code{-h}
-@cindex @code{-?}
+@cindex @sortas{options, Octave command --help} @ @ @code{--help}
+@cindex @sortas{options, Octave command -h} @ @ @code{-h}
 Print short help message and exit.
 
 @item --image-path @var{path}
-@cindex @code{--image-path @var{path}}
+@cindex @sortas{options, Octave command --image-path} @ @ @code{--image-path @var{path}}
 Add path to the head of the search path for images.  The value of
 @var{path} specified on the command line will override any value of
 @w{@env{OCTAVE_IMAGE_PATH}} found in the environment, but not any commands
 in the system or user startup files that set the built-in variable
 @w{@env{IMAGE_PATH}}.
 
 @item --info-file @var{filename}
-@cindex @code{--info-file @var{filename}}
+@cindex @sortas{options, Octave command --info-file} @ @ @code{--info-file @var{filename}}
 Specify the name of the info file to use.  The value of @var{filename}
 specified on the command line will override any value of
 @w{@env{OCTAVE_INFO_FILE}} found in the environment, but not any commands
 in the system or user startup files that use the @code{info_file}
 function.
 
 @item --info-program @var{program}
-@cindex @code{--info-program @var{program}}
+@cindex @sortas{options, Octave command --info-program} @ @ @code{--info-program @var{program}}
 Specify the name of the info program to use.  The value of @var{program}
 specified on the command line will override any value of
 @w{@env{OCTAVE_INFO_PROGRAM}} found in the environment, but not any
 commands in the system or user startup files that use the
 @code{info_program} function.
 
 @item  --interactive
 @itemx -i
-@cindex @code{--interactive}
-@cindex @code{-i}
+@cindex @sortas{options, Octave command --interactive} @ @ @code{--interactive}
+@cindex @sortas{options, Octave command -i} @ @ @code{-i}
 Force interactive behavior.  This can be useful for running Octave via a
 remote shell command or inside an Emacs shell buffer.
 
 @item --jit-compiler
-@cindex @code{--jit-compiler}
+@cindex @sortas{options, Octave command --jit-compiler} @ @ @code{--jit-compiler}
 Enable the JIT compiler used for accelerating loops.
 
 @item --line-editing
-@cindex @code{--line-editing}
+@cindex @sortas{options, Octave command --line-editing} @ @ @code{--line-editing}
 Force readline use for command-line editing.
 
 @item --no-gui
-@cindex @code{--no-gui}
+@cindex @sortas{options, Octave command --no-gui} @ @ @code{--no-gui}
 Disable the graphical user interface (GUI) and use the command line
 interface (CLI) instead.
 
 @item  --no-history
 @itemx -H
-@cindex @code{--no-history}
-@cindex @code{-H}
+@cindex @sortas{options, Octave command --no-history} @ @ @code{--no-history}
+@cindex @sortas{options, Octave command -H} @ @ @code{-H}
 Disable recording of command-line history.
 
 @item --no-init-file
-@cindex @code{--no-init-file}
+@cindex @sortas{options, Octave command --no-init-file} @ @ @code{--no-init-file}
+@cindex @sortas{octaverc ~/.octaverc} @code{~/.octaverc}
+@cindex @sortas{octaverc .octaverc} @code{.octaverc}
 Don't read the initialization files @file{~/.octaverc} and @file{.octaverc}.
 
 @item --no-init-path
-@cindex @code{--no-init-path}
+@cindex @sortas{options, Octave command --no-init-path} @ @ @code{--no-init-path}
 Don't initialize the search path for function files to include default
 locations.
 
 @item --no-line-editing
-@cindex @code{--no-line-editing}
+@cindex @sortas{options, Octave command --no-line-editing} @ @ @code{--no-line-editing}
 Disable command-line editing.
 
 @item --no-site-file
-@cindex @code{--no-site-file}
+@cindex @sortas{options, Octave command --no-site-file} @ @ @code{--no-site-file}
+@cindex site startup file
 Don't read the site-wide @file{octaverc} initialization files.
 
 @item  --no-window-system
 @itemx -W
-@cindex @code{--no-window-system}
+@cindex @sortas{options, Octave command --no-window-system} @ @ @code{--no-window-system}
+@cindex @sortas{options, Octave command -W} @ @ @code{-W}
 Disable use of a windowing system including graphics.  This forces a
 strictly terminal-only environment.
 
 @item  --norc
 @itemx -f
-@cindex @code{--norc}
-@cindex @code{-f}
+@cindex @sortas{options, Octave command --norc} @ @ @code{--norc}
+@cindex @sortas{options, Octave command -f} @ @ @code{-f}
 Don't read any of the system or user initialization files at startup.
 This is equivalent to using both of the options @option{--no-init-file}
 and @option{--no-site-file}.
 
 @item  --path @var{path}
 @itemx -p @var{path}
-@cindex @code{--path @var{path}}
-@cindex @code{-p @var{path}}
+@cindex @sortas{options, Octave command --path} @ @ @code{--path @var{path}}
+@cindex @sortas{options, Octave command -p} @ @ @code{-p @var{path}}
 Add path to the head of the search path for function files.  The
 value of @var{path} specified on the command line will override any value
 of @w{@env{OCTAVE_PATH}} found in the environment, but not any commands in the
 system or user startup files that set the internal load path through one
 of the path functions.
 
 @item --persist
-@cindex @code{--persist}
+@cindex @sortas{options, Octave command --persist} @ @ @code{--persist}
 Go to interactive mode after @option{--eval} or reading from a file
 named on the command line.
 
 @item  --silent
 @itemx --quiet
 @itemx -q
-@cindex @code{--silent}
-@cindex @code{--quiet}
-@cindex @code{-q}
+@cindex @sortas{options, Octave command --silent} @ @ @code{--silent}
+@cindex @sortas{options, Octave command --quie} @ @ @code{--quiet}
+@cindex @sortas{options, Octave command -q} @ @ @code{-q}
 Don't print the usual greeting and version message at startup.
 
 @item --texi-macros-file @var{filename}
-@cindex @code{--texi-macros-file @var{filename}}
+@cindex @sortas{options, Octave command --texi-macros-file} @ @ @code{--texi-macros-file @var{filename}}
 Specify the name of the file containing Texinfo macros for use by makeinfo.
 
 @item  --traditional
 @itemx --braindead
-@cindex @code{--traditional}
-@cindex @code{--braindead}
+@cindex @sortas{options, Octave command --traditional} @ @ @code{--traditional}
+@cindex @sortas{options, Octave command --braindead} @ @ @code{--braindead}
 For compatibility with @sc{matlab}, set initial values for
 user preferences to the following values
 
 @example
 @group
 PS1                             = ">> "
 PS2                             = ""
 beep_on_error                   = true
@@ -274,24 +277,24 @@ Octave:possible-matlab-short-circuit-ope
 @noindent
 Note that this does not enable the @code{Octave:language-extension}
 warning, which you might want if you want to be told about writing code
 that works in Octave but not @sc{matlab} (@pxref{XREFwarning,,warning},
 @ref{XREFwarning_ids,,warning_ids}).
 
 @item  --verbose
 @itemx -V
-@cindex @code{--verbose}
-@cindex @code{-V}
+@cindex @sortas{options, Octave command --verbose} @ @ @code{--verbose}
+@cindex @sortas{options, Octave command -V} @ @ @code{-V}
 Turn on verbose output.
 
 @item  --version
 @itemx -v
-@cindex @code{--version}
-@cindex @code{-v}
+@cindex @sortas{options, Octave command --version} @ @ @code{--version}
+@cindex @sortas{options, Octave command -v} @ @ @code{-v}
 Print the program version number and exit.
 
 @item @var{file}
 Execute commands from @var{file}.  Exit when done unless
 @option{--persist} is also specified.
 @end table
 
 Octave also includes several functions which return information
@@ -353,33 +356,34 @@ is the version number of Octave.  This f
 to the default Octave environment can be made globally for all users of
 a particular version of Octave.  Care should be taken when making
 changes to this file since all users of Octave at your site will be
 affected.  The default file may be overridden by the environment variable
 @w{@env{OCTAVE_VERSION_INITFILE}}.
 
 @item ~/.octaverc
 @cindex personal startup file
-@cindex @code{~/.octaverc}
+@cindex @sortas{octaverc ~/.octaverc} @code{~/.octaverc}
 This file is used to make personal changes to the default
 Octave environment.
 
 @item .octaverc
 @cindex project startup file
-@cindex @code{.octaverc}
+@cindex @sortas{octaverc .octaverc} @code{.octaverc}
 This file can be used to make changes to the default Octave environment
 for a particular project.  Octave searches for this file in the current
 directory after it reads @file{~/.octaverc}.  Any use of the @code{cd}
 command in the @file{~/.octaverc} file will affect the directory where
 Octave searches for @file{.octaverc}.
 
 If you start Octave in your home directory, commands from the file
 @file{~/.octaverc} will only be executed once.
 
 @item startup.m
+@cindex @code{startup.m}
 This file is used to make personal changes to the default
 Octave environment.  It is executed for @sc{matlab} compatibility, but
 @file{~/.octaverc} is the preferred location for configuration changes.
 @end table
 
 A message will be displayed as each of the startup files is read if you
 invoke Octave with the @option{--verbose} option but without the
 @option{--silent} option.
@@ -742,17 +746,17 @@ is saved.
 @DOCSTRING(history_size)
 
 @DOCSTRING(history_timestamp_format_string)
 
 @DOCSTRING(EDITOR)
 
 @node Customizing readline
 @subsection Customizing @code{readline}
-@cindex @code{~/.inputrc}
+@cindex @sortas{inputrc ~/.inputrc} @code{~/.inputrc}
 @cindex customizing @code{readline}
 @cindex @code{readline} customization
 
 Octave uses the GNU Readline library for command-line editing and
 history features.  Readline is very flexible and can be modified through
 a configuration file of commands (See the GNU Readline library for the
 exact command syntax).  The default configuration file is normally
 @file{~/.inputrc}.
@@ -932,18 +936,18 @@ error before trying again.
 
 @node Executable Octave Programs
 @section Executable Octave Programs
 @cindex executable scripts
 @cindex scripts
 @cindex batch processing
 @cindex self contained programs
 @cindex program, self contained
-@cindex @samp{#!}
 
+@opindex @code{#!} self-contained script
 Once you have learned Octave, you may want to write self-contained
 Octave scripts, using the @samp{#!} script mechanism.  You can do this
 on GNU systems and on many Unix systems @footnote{The @samp{#!}
 mechanism works on Unix systems derived from Berkeley Unix, System V
 Release 4, and some System V Release 3 systems.}.
 
 Self-contained Octave scripts are useful when you want to write a
 program which users can invoke without knowing that the program is
@@ -1056,19 +1060,19 @@ typically hard to understand without the
 @menu
 * Single Line Comments::
 * Block Comments::
 * Comments and the Help System::
 @end menu
 
 @node Single Line Comments
 @subsection Single Line Comments
-@cindex @samp{#}
-@cindex @samp{%}
 
+@opindex @code{#} comment marker
+@opindex @code{%} comment marker
 In the Octave language, a comment starts with either the sharp sign
 character, @samp{#}, or the percent symbol @samp{%} and continues to the
 end of the line.  Any text following the sharp sign or percent symbol is
 ignored by the Octave interpreter and not executed.  The following example
 shows whole line and partial line comments.
 
 @example
 @group
@@ -1081,19 +1085,19 @@ function countdown
 endfunction
 @end group
 @end example
 
 @node Block Comments
 @subsection Block Comments
 @cindex block comments
 @cindex multi-line comments
-@cindex @samp{#@{}
-@cindex @samp{%@{}
 
+@opindex @code{#@{} block comment marker
+@opindex @code{%@{} block comment marker
 Entire blocks of code can be commented by enclosing the code between
 matching @samp{#@{} and @samp{#@}} or @samp{%@{} and @samp{%@}} markers.
 For example,
 
 @example
 @group
 function quick_countdown
   # Count down for main rocket engines
diff --git a/doc/interpreter/expr.txi b/doc/interpreter/expr.txi
--- a/doc/interpreter/expr.txi
+++ b/doc/interpreter/expr.txi
@@ -101,17 +101,17 @@ row of the matrix.
 @group
 a(1, [1, 2])  # row 1, columns 1 and 2
 a(1, 1:2)     # row 1, columns in range 1-2
 a(1, :)       # row 1, all columns
 @end group
 @end example
 
 @cindex @code{end}, indexing
-@cindex :end
+@cindex @sortas{end:} @code{end:} and @code{:end}
 
 In index expressions the keyword @code{end} automatically refers to
 the last entry for a particular dimension.  This magic index can also
 be used in ranges and typically eliminates the needs to call
 @code{size} or @code{length} to gather array bounds before indexing.
 For example:
 
 @example
diff --git a/doc/interpreter/stmt.txi b/doc/interpreter/stmt.txi
--- a/doc/interpreter/stmt.txi
+++ b/doc/interpreter/stmt.txi
@@ -862,19 +862,18 @@ end_try_catch
 will automatically store the output of @code{lasterror} in the structure
 @var{err}.  @xref{Errors and Warnings}, for more information about the
 @code{lasterr} and @code{lasterror} functions.
 
 
 @node Continuation Lines
 @section Continuation Lines
 @cindex continuation lines
-@cindex @code{...} continuation marker
-@cindex @code{\} continuation marker
 
+@opindex @code{...} continuation marker
 In the Octave language, most statements end with a newline character and
 you must tell Octave to ignore the newline character in order to
 continue a statement from one line to the next.  Lines that end with the
 characters @code{...} are joined with the following line before they are
 divided into tokens by Octave's parser.  For example, the lines
 
 @example
 @group
@@ -896,16 +895,17 @@ x = long_variable_name ...    # comment 
     + longer_variable_name ...comment two
     - 42                      # last comment
 @end group
 @end example
 
 @noindent
 is equivalent to the one shown above.
 
+@opindex @code{\} continuation marker
 Inside double-quoted string constants, the character @code{\} has to be
 used as continuation marker.  The @code{\} must appear at the end of the
 line just before the newline character:
 
 @example
 @group
 s = "This text starts in the first line \
 and is continued in the second line."
diff --git a/scripts/plot/appearance/rticks.m b/scripts/plot/appearance/rticks.m
--- a/scripts/plot/appearance/rticks.m
+++ b/scripts/plot/appearance/rticks.m
@@ -13,17 +13,17 @@
 ## GNU General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} rticks
-## @deftypefnx {} rticks (@var{tickval})
+## @deftypefnx {} {} rticks (@var{tickval})
 ## @deftypefnx {} {@dots{} =} rticks (@var{hax}, @dots{})
 ## Query or set the tick values on the r-axis of the current axis.
 ##
 ## When called without argument, return the current tick locations as specified
 ## in the @qcode{"rtick"} axes property.  These locations can be changed by
 ## calling @code{rticks} with a vector of tick values.  Note: ascending order
 ## is not required.
 ##
diff --git a/scripts/plot/appearance/thetaticks.m b/scripts/plot/appearance/thetaticks.m
--- a/scripts/plot/appearance/thetaticks.m
+++ b/scripts/plot/appearance/thetaticks.m
@@ -13,17 +13,17 @@
 ## GNU General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} thetaticks
-## @deftypefnx {} thetaticks (@var{tickval})
+## @deftypefnx {} {} thetaticks (@var{tickval})
 ## @deftypefnx {} {@dots{} =} thetaticks (@var{hax}, @dots{})
 ## Query or set the tick values on the theta-axis of the current axis.
 ##
 ## When called without argument, return the current tick locations as specified
 ## in the @qcode{"ttick"} axes property.  These locations can be changed by
 ## calling @code{thetaticks} with a vector of tick values.  Note: ascending
 ## order is not required.
 ##
diff --git a/scripts/plot/appearance/xticklabels.m b/scripts/plot/appearance/xticklabels.m
--- a/scripts/plot/appearance/xticklabels.m
+++ b/scripts/plot/appearance/xticklabels.m
@@ -14,19 +14,19 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} xticklabels
 ## @deftypefnx {} {@var{mode} =} xticklabels ("mode")
-## @deftypefnx {} xticklabels (@var{tickval})
-## @deftypefnx {} xticklabels ("auto")
-## @deftypefnx {} xticklabels ("manual")
+## @deftypefnx {} {} xticklabels (@var{tickval})
+## @deftypefnx {} {} xticklabels ("auto")
+## @deftypefnx {} {} xticklabels ("manual")
 ## @deftypefnx {} {@dots{} =} xticklabels (@var{hax}, @dots{})
 ## Query or set the tick labels on the x-axis of the current axis.
 ##
 ## When called without an argument, return a cell array of strings of the
 ## current tick labels as specified in the @qcode{"xticklabel"} axes property.
 ## These labels can be changed by calling @code{xticklabels} with a cell array
 ## of strings.  Note: a vector of numbers will be mapped to a cell array of
 ## strings.  If fewer labels are specified than the current number of ticks,
diff --git a/scripts/plot/appearance/xticks.m b/scripts/plot/appearance/xticks.m
--- a/scripts/plot/appearance/xticks.m
+++ b/scripts/plot/appearance/xticks.m
@@ -14,19 +14,19 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} xticks
 ## @deftypefnx {} {@var{mode} =} xticks ("mode")
-## @deftypefnx {} xticks (@var{tickval})
-## @deftypefnx {} xticks ("auto")
-## @deftypefnx {} xticks ("manual")
+## @deftypefnx {} {} xticks (@var{tickval})
+## @deftypefnx {} {} xticks ("auto")
+## @deftypefnx {} {} xticks ("manual")
 ## @deftypefnx {} {@dots{} =} xticks (@var{hax}, @dots{})
 ## Query or set the tick values on the x-axis of the current axis.
 ##
 ## When called without an argument, return the current tick locations as
 ## specified in the @qcode{"xtick"} axes property.  These locations can be
 ## changed by calling @code{xticks} with a vector of tick values.  Note:
 ## ascending order is not required.
 ##
diff --git a/scripts/plot/appearance/yticklabels.m b/scripts/plot/appearance/yticklabels.m
--- a/scripts/plot/appearance/yticklabels.m
+++ b/scripts/plot/appearance/yticklabels.m
@@ -14,19 +14,19 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} yticklabels
 ## @deftypefnx {} {@var{mode} =} yticklabels ("mode")
-## @deftypefnx {} yticklabels (@var{tickval})
-## @deftypefnx {} yticklabels ("auto")
-## @deftypefnx {} yticklabels ("manual")
+## @deftypefnx {} {} yticklabels (@var{tickval})
+## @deftypefnx {} {} yticklabels ("auto")
+## @deftypefnx {} {} yticklabels ("manual")
 ## @deftypefnx {} {@dots{} =} yticklabels (@var{hax}, @dots{})
 ## Query or set the tick labels on the x-axis of the current axis.
 ##
 ## When called without an argument, return a cell array of strings of the
 ## current tick labels as specified in the @qcode{"yticklabel"} axes property.
 ## These labels can be changed by calling @code{yticklabels} with a cell array
 ## of strings.  Note: a vector of numbers will be mapped to a cell array of
 ## strings.  If fewer labels are specified than the current number of ticks,
diff --git a/scripts/plot/appearance/yticks.m b/scripts/plot/appearance/yticks.m
--- a/scripts/plot/appearance/yticks.m
+++ b/scripts/plot/appearance/yticks.m
@@ -14,19 +14,19 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} yticks
 ## @deftypefnx {} {@var{mode} =} yticks ("mode")
-## @deftypefnx {} yticks (@var{tickval})
-## @deftypefnx {} yticks ("auto")
-## @deftypefnx {} yticks ("manual")
+## @deftypefnx {} {} yticks (@var{tickval})
+## @deftypefnx {} {} yticks ("auto")
+## @deftypefnx {} {} yticks ("manual")
 ## @deftypefnx {} {@dots{} =} yticks (@var{hax}, @dots{})
 ## Query or set the tick values on the y-axis of the current axis.
 ##
 ## When called without an argument, return the current tick locations as
 ## specified in the @qcode{"ytick"} axes property.  These locations can be
 ## changed by calling @code{yticks} with a vector of tick values.  Note:
 ## ascending order is not required.
 ##
diff --git a/scripts/plot/appearance/zticklabels.m b/scripts/plot/appearance/zticklabels.m
--- a/scripts/plot/appearance/zticklabels.m
+++ b/scripts/plot/appearance/zticklabels.m
@@ -14,19 +14,19 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} zticklabels
 ## @deftypefnx {} {@var{mode} =} zticklabels ("mode")
-## @deftypefnx {} zticklabels (@var{tickval})
-## @deftypefnx {} zticklabels ("auto")
-## @deftypefnx {} zticklabels ("manual")
+## @deftypefnx {} {} zticklabels (@var{tickval})
+## @deftypefnx {} {} zticklabels ("auto")
+## @deftypefnx {} {} zticklabels ("manual")
 ## @deftypefnx {} {@dots{} =} zticklabels (@var{hax}, @dots{})
 ## Query or set the tick labels on the x-axis of the current axis.
 ##
 ## When called without an argument, return a cell array of strings of the
 ## current tick labels as specified in the @qcode{"zticklabel"} axes property.
 ## These labels can be changed by calling @code{zticklabels} with a cell array
 ## of strings.  Note: a vector of numbers will be mapped to a cell array of
 ## strings.  If fewer labels are specified than the current number of ticks,
diff --git a/scripts/plot/appearance/zticks.m b/scripts/plot/appearance/zticks.m
--- a/scripts/plot/appearance/zticks.m
+++ b/scripts/plot/appearance/zticks.m
@@ -14,19 +14,19 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {} {@var{tickval} =} zticks
 ## @deftypefnx {} {@var{mode} =} zticks ("mode")
-## @deftypefnx {} zticks (@var{tickval})
-## @deftypefnx {} zticks ("auto")
-## @deftypefnx {} zticks ("manual")
+## @deftypefnx {} {} zticks (@var{tickval})
+## @deftypefnx {} {} zticks ("auto")
+## @deftypefnx {} {} zticks ("manual")
 ## @deftypefnx {} {@dots{} =} zticks (@var{hax}, @dots{})
 ## Query or set the tick values on the z-axis of the current axis.
 ##
 ## When called without an argument, return the current tick locations as
 ## specified in the @qcode{"ztick"} axes property.  These locations can be
 ## changed by calling @code{zticks} with a vector of tick values.  Note:
 ## ascending order is not required.
 ##
